pinot-common/src/main/java/org/apache/pinot/common/utils/DataTable.java

```diff
@@ -35,6 +35,8 @@
   String NUM_SEGMENTS_QUERIED = "numSegmentsQueried";
   String NUM_SEGMENTS_PROCESSED = "numSegmentsProcessed";
   String NUM_SEGMENTS_MATCHED = "numSegmentsMatched";
+  String NUM_CONSUMING_SEGMENTS_QUERIED = "numConsumingQueried";
+  String MIN_CONSUMING_TIMESTAMP = "minConsumingTs";
```

Comment:
I need a better name for this - its meant to indicate the min latest ingestion timestamp across segments.